hack/make.sh

```diff
@@ -26,7 +26,7 @@ set -o pipefail
 export DOCKER_PKG='github.com/docker/docker'
 export SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 export MAKEDIR="$SCRIPTDIR/make"
-export PKG_CONFIG=${PKG_CONFIG:-pkg-config}
+export PKG_CONFIG=${PKG_CONFIG:-$(go env PKG_CONFIG)}
```

Comment:
`PKG_CONFIG` is not a standard part of `go env` -- is the mechanism XX uses to set it really just setting it in the `goenv` file? Why doesn't a regular fall-through (xx exporting it) work here?

Reply:
It seems to: https://github.com/golang/go/blob/245e95dfabd77f337373bf2d6bb47cd353ad8d74/src/internal/cfg/cfg.go#L67

Reply:
Hmm, by standard I guess I mean "set by default" -- I'm not entirely familiar with how things are invoked.

Looking a little further, it seems like xx is replacing the go compiler with a shell script wrapper -- so to get an environment variable from xx, we need to interrogate it instead of letting it bubble down through the process tree?